app/components/layout/sidebar/index.tsx

```diff
@@ -48,68 +46,114 @@ export default function Sidebar({ user }: Props) {
-              <NavLink
-                className={({ isActive }) =>
-                  tw(
-                    "my-1 flex items-center gap-3 rounded-md px-3 py-2.5 text-text-md font-semibold text-gray-700 transition-all duration-75 hover:bg-gray-100 hover:text-gray-900",
-                    isActive ? "bg-gray-100 text-gray-900" : ""
-                  )
-                }
-                to={item.to}
-                data-test-id={`${item.label.toLowerCase()}SidebarMenuItem`}
-              >
-                <i className="icon text-gray-500">{item.icon}</i>
-                <span className="text whitespace-nowrap transition duration-200 ease-linear">
-                  {item.label}
-                </span>
-              </NavLink>
-            </li>
-          ))}
-        </ul>
-      </div>
-      <div className="mt-auto">
-        <SidebarBottom
-          user={{
-            ...user,
-          }}
-        />
-      </div>
-    </aside>
+      </header>
+      <div
+        onClick={toggleSidebar}
+        className={tw(
+          "fixed right-0 top-0 z-10 h-screen w-screen cursor-pointer bg-gray-25/10 backdrop-blur transition duration-300 ease-in-out md:hidden",
+          isSidebarCollapsed ? "invisible opacity-0" : "visible"
+        )}
+      ></div>
+      <aside
+        id="main-navigation"
+        ref={mainNavigationRef}
+        onMouseEnter={toggleSidebar}
+        onMouseLeave={toggleSidebar}
+        className={tw(
+          `fixed left-[-100%] top-0 z-30 flex h-screen w-80 flex-col border-r border-gray-200 bg-white p-4 shadow-[0px_20px_24px_-4px_rgba(16,24,40,0.08),_0px_8px_8px_-4px_rgba(16,24,40,0.03)] transition-all duration-300 ease-linear md:sticky md:left-0 md:px-6 md:py-8 md:duration-200`,
+          isSidebarCollapsed
+            ? "collapsed-navigation w-[92px] overflow-hidden"
+            : "left-0 w-[270px] min-[350px]:w-[312px]"
+        )}
+      >
+        <div className="navigation-header flex items-center justify-between">
+          <Link to="." title="Home" className="flex items-center">
+            <img
+              src="/images/shelf-symbol.png"
+              alt="Shelf Logo"
+              className="mx-1.5 inline h-[32px]"
+            />
+            <span className="logo-text transition duration-200 ease-linear">
+              <ShelfTypography />
+            </span>
+          </Link>
+          <button
+            className="hidden transition-all duration-200 ease-linear md:block"
+            onClick={manageUncollapsedSidebar}
+          >
+            <i className="icon text-gray-500">
+              {maintainUncollapsedSidebar ? (
+                <ActiveSwitchIcon />
+              ) : (
+                <SwitchIcon />
+              )}
+            </i>
+          </button>
+        </div>
+        <div className="flex-1">
+          <ul className="menu mt-6 md:mt-10">
+            {menuItems.map((item) => (
+              <li key={item.label}>
+                <NavLink
```

Comment:
Here we need to close the menu on click on mobile. So if the user navigates they actually see the page and dont have to manually close the menu.